src/main/kotlin/baseball/view/View.kt

```diff
@@ -0,0 +1,39 @@
+package baseball.view
+
+import baseball.domain.RoundResult
+
+class View {
+
+    fun printStartGameMessage() {
+        println("숫자 야구 게임을 시작합니다.")
+    }
+
+    fun inputUserNumbers(): String {
+        print("숫자를 입력해주세요 : ")
+        return readln();
+    }
+
+    fun printRoundResult(roundResult: RoundResult) {
+        if (roundResult.isNothing()) {
+            println("낫싱")
+        }
+        if(roundResult.isOnlyBall()) {
+            println("%d볼".format(roundResult.ballCount))
```

Comment:
format 을 사용하다니 당신은 자바 개발자!

```suggestion
            println("${roundResult.ballCount}볼")
```

Reply:
헤헤 감사합니다